stl/inc/memory

```diff
@@ -1145,6 +1145,12 @@ public:
     _Ref_count_resource(_Resource _Px, _Dx _Dt)
         : _Ref_count_base(), _Mypair(_One_then_variadic_args_t{}, _STD move(_Dt), _Px) {}
 
+#ifdef __EDG__ // TRANSITION, VSO-1292293
+    virtual ~_Ref_count_resource() noexcept override {} // TRANSITION, should be non-virtual

```

Comment:
I find the `virtual override` to be a code smell. A function that is marked as override *must* be virtual. So this is only adding noise

Reply:
That’s a reasonable point, and it’s tracked by #303. In this PR, I’m remaining consistent with the current convention; if and when we decide to change it, we should do so globally.